From 5eb0f49b0a4ca29ed8cb3d05b59076551943ac9e Mon Sep 17 00:00:00 2001
From: "Lori A. Burns" <lori.burns@gmail.com>
Date: Thu, 1 Jun 2023 03:29:28 -0400
Subject: [PATCH] win: dll to bin, no pkgconfig

---
 CMakeLists.txt     | 8 +++++---
 lib/CMakeLists.txt | 2 +-
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c37234a..4caa7c5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,9 +88,11 @@ endif()
 
 include(CTest)
 
-configure_file("libcerf.pc.in" "libcerf.pc" @ONLY)
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libcerf.pc"
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig/")
+if(NOT MSVC)
+    configure_file("libcerf.pc.in" "libcerf.pc" @ONLY)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libcerf.pc"
+        DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig/")
+endif()
 
 add_subdirectory(lib)
 add_subdirectory(test)
diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index 8ffe6fb..ab63028 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -39,7 +39,7 @@ install(
     TARGETS ${lib}
     EXPORT ${intf}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     COMPONENT Libraries)
 install(
-- 
2.40.1

